wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -1019,12 +1022,7 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
         val firstIdentityKey = getBlockchainIdentityKey(0, encryptionKey) ?: return null
 
         return try {
-            val identityBytes = platform.client.getIdentityByFirstPublicKey(firstIdentityKey.pubKeyHash, true)
-            if (identityBytes != null && identityBytes.isNotEmpty()) {
-                platform.dpp.identity.createFromBuffer(identityBytes)
-            } else {
-                null
-            }
+            platform.stateRepository.fetchIdentityFromPubKeyHash(firstIdentityKey.pubKeyHash)
```

Comment:
as in Android DashPay Library, `platform.stateRepository.` is the better way to make platform calls, so the "intercepter" will catch the calls.